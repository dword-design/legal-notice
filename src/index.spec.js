import * as self from './index.js'

export default {
  de() {
    expect(self.de).toMatchSnapshot(this)
  },
  en() {
    expect(self.en).toMatchSnapshot(this)
  },
}
