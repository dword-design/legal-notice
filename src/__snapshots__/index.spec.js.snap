// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index de 1`] = `
"<h2>Impressum</h2>
<p>
  Sebastian Landwehr<br>
  Breite Str. 20<br>
  53111 Bonn<br>
  Deutschland
</p>

<h3>Kontakt</h3>
<p>
  E-Mail: info@sebastianlandwehr.com<br>
  Telefon: +49 177 5055641<br>
</p>

<h3>Redaktionell verantwortlich gemäß § 55 Abs. 2 RStV</h3>
<p>
  Sebastian Landwehr<br>
  Breite Str. 20<br>
  53111 Bonn<br>
  Deutschland
</p>"
`;

exports[`index en 1`] = `
"<h2>Legal Notice</h2>
<p>
  Sebastian Landwehr<br>
  Breite Str. 20<br>
  53111 Bonn<br>
  Germany
</p>

<h3>Contact</h3>
<p>
  Email: info@sebastianlandwehr.com<br>
  Phone: +49 177 5055641<br>
</p>

<h3>Person responsible for content in accordance with § 55 Abs. 2 RStV</h3>
<p>
  Sebastian Landwehr<br>
  Breite Str. 20<br>
  53111 Bonn<br>
  Germany
</p>"
`;
